src/main/java/com/exadel/discount/repository/CityRepository.java

```diff
@@ -0,0 +1,20 @@
+package com.exadel.discount.repository;
+
+import com.exadel.discount.entity.City;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@Repository
+public interface CityRepository extends JpaRepository<City, UUID> {
+    @Query(nativeQuery = true,
```

Comment:
Don't use native query if you really need it

Reply:
Done